src/main/java/com/microsoft/aad/msal4j/AcquireTokenByInteractiveFlowSupplier.java

```diff
@@ -34,9 +34,35 @@ class AcquireTokenByInteractiveFlowSupplier extends AuthenticationResultSupplier
 
     @Override
     AuthenticationResult execute() throws Exception {
-        AuthorizationResult authorizationResult = getAuthorizationResult();
-        validateState(authorizationResult);
-        return acquireTokenWithAuthorizationCode(authorizationResult);
+        if (clientApplication.allowBroker()) {
```

Comment:
We should have boolean getters prefixed with `is`, so perhaps this method should be named `isBrokerAllowed()` or similar?

Reply:
This is just the flag decided on in the [design doc](https://identitydivision.visualstudio.com/DevEx/_git/AuthLibrariesApiReview?path=/%5BXplat%5D%20MSALRuntime/Windows%20MSALRuntime/IntegrationWithMsalRuntime.md&version=GCd5b0360363a21b7c812d6303c911f1c420117dcc&_a=preview) (mainly according to conventions in Python and .NET since they were the frontrunners on this). Normally I'd agree that it's weird and change it to something like 'isBrokerAllowed', but consistency across MSALs is a focus on this project, and we other parameters like this in our library already so it shouldn't be too confusing.